src/components/gutter-control/gutter-wrapper.php

```diff
@@ -33,10 +33,60 @@ function coblocks_add_gutter_styles( $styles, $attributes ) {
  * @return array
  */
 function coblocks_add_gutter_class( $classes, $attributes ) {
-	if ( isset( $attributes['columns'] ) && isset( $attributes['gutter'] ) ) {
+	if ( isset( $attributes['columns'] ) && 1 < intval( $attributes['columns'] ) && isset( $attributes['gutter'] ) ) {
 		array_push( $classes, 'has-' . $attributes['gutter'] . '-gutter' );
 	}
 
 	return $classes;
 }
 add_filter( 'coblocks_render_wrapper_class', 'coblocks_add_gutter_class', 10, 2 );
+
+/**
+ * Add gutter attribute in php
+ *
+ * @param WP_Block_Type $block_type Block Type.
+ */
+function coblocks_register_gutter_support( $block_type ) {
+	$has_gutter_support = false;
+
+	if ( property_exists( $block_type, 'supports' ) ) {
+
+		$has_gutter_support = coblocks_experimental_get( $block_type->supports, array( 'gutter' ), false );
+
+	}
+
+	if ( $has_gutter_support ) {
```

Comment:
@AnthonyLedesma I would avoid the big if condition here and reverse and return early.
```php
if (  ! $has_gutter_support ) {
  return
}
```